### PR TITLE
Add handling logic for custom theme files

### LINKED ISSUE #71: Custom themes aren't loaded
**Describe the bug**
Setting
```md
---
theme: ./theme.json
---
```
results in the theme not being loaded and instead the default theme is used.

**To Reproduce**
1. Create `presentation.md` with the following content
```
---
theme: ./theme.json
---

# Heading 1
## Heading 2
```
2. `wget https://github.com/maaslalani/slides/raw/main/styles/theme.json`
3. Change the `h1.prefix` to `>>`
4. Run `slides presentation.md` and see that the modified theme isn't loaded

**Expected behavior**
The specified theme file should be loaded

**Additional context**
Looks like the code doesn't handle this case? I feel like [this function](https://github.com/maaslalani/slides/blob/5a8e991163707ad41f0bd4192c36db9093ff733c/styles/styles.go#L50) is missing any logic to handle a custom theme file?

### PR DESCRIPTION
**Pull request by itmecho**
Fixes #71 

### Changes Introduced
Tries to read theme files. On success, the theme will be loaded and applied. On failure, the previous behaviour will take effect.

### CODE REVIEW

File: styles/styles.go [hunk 1]
```diff
@@ -58,13 +59,19 @@ func SelectTheme(theme string) glamour.TermRendererOption {
...
-		if !termenv.HasDarkBackground(){
-			return glamour.WithStyles(glamour.LightStyleConfig)
+			if !termenv.HasDarkBackground() {
+				return glamour.WithStyles(glamour.LightStyleConfig)
+			}
+			return glamour.WithStylesFromJSONBytes(DefaultTheme)
 		}
-		return glamour.WithStylesFromJSONBytes(DefaultTheme)
+
+		return glamour.WithStylesFromJSONBytes(bytes)
```
Comment (maaslalani) [inline styles/styles.go:75]: We should just return this if `err != nil` to remove a layer of nesting

**Review by maaslalani — COMMENTED** [review]
Thanks so so much ❤️ You're awesome for fixing this!

File: styles/styles.go (hunk 1 above)
Comment (itmecho) [reply styles/styles.go]: Done

File: styles/styles.go [hunk 2]
```diff
@@ -58,13 +59,20 @@ func SelectTheme(theme string) glamour.TermRendererOption {
 	case "notty":
 		return glamour.WithStyles(glamour.NoTTYStyleConfig)
 	default:
+		bytes, err := os.ReadFile(theme)
+		if err == nil {
+			return glamour.WithStylesFromJSONBytes(bytes)
+		}
+		// Should log a warning so the user knows we failed to read their theme file
```
Comment (maaslalani) [inline styles/styles.go:66]: 👍  to this, I'm thinking about writing all error messages to a log file because we fail silently in most cases

**Comment by maaslalani** [comment]
Thanks so much! ❤️  Super appreciated!

**Comment by itmecho** [comment]
You're welcome! Thanks for creating the project! :D

**Pull request merged**